barretenberg/cpp/src/barretenberg/vm/avm/recursion/recursive_flavor.hpp

```diff
@@ -25,6 +25,9 @@ template <typename BuilderType> class AvmRecursiveFlavor_ {
 
     using Relations = AvmFlavor::Relations_<FF>;
 
+    // indicates when evaluating sumcheck, edges must be extended to be MAX_TOTAL_RELATION_LENGTH
+    static constexpr bool USE_SHORT_MONOMIALS = false;
```

Comment:
Would it make sense to align it with the native flavor?
What I mean is to set it:
`    static constexpr bool USE_SHORT_MONOMIALS = NativeFlavor::USE_SHORT_MONOMIALS;
`

Reply:
good call, updated